src/Dapr.Client/DaprClient.cs

```diff
@@ -92,14 +92,14 @@ public static HttpClient CreateInvokeHttpClient(string appId = null, string dapr
             return httpClient;
         }
 
-        internal static KeyValuePair<string, string> GetDaprApiTokenHeader(string apiToken)
+        internal static KeyValuePair<string, string>? GetDaprApiTokenHeader(string apiToken)
```

Comment:
this is clearer to model it as a nullable since the api token is optional.

Internal api so there's no breaking change impact